.github/workflows/test-generated-project.yml

```
@@ -39,17 +39,12 @@ jobs:
       - name: Generate project
         run: . ./scripts/generateAdvancedAWS.sh
 
-      - name: Install Terraform
-        uses: hashicorp/setup-terraform@v2
-        with:
-          terraform_version: ${{ env.TERRAFORM_VERSION }}
+      - name: Install dependencies in .tool-versions
+        uses: asdf-vm/actions/install@v2
 
       - name: Run Terraform format
         run: terraform fmt -recursive -check
 
-      - name: Run tfsec linter
-        id: tfsec
-        uses: aquasecurity/tfsec-action@v1.0.3
-        with:
-          version: ${{ env.TFSEC_VERSION }}
-
+      - name: Run trivy linter
```

```suggestion
      - name: Run trivy scanner
```

According to the doc, Trivy is more referred to as a scanner, rather than a linter 💭